<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Form</class>
    <widget class="QWidget" name="Form">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>640</width>
                <height>132</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <layout class="QGridLayout" name="gridLayout">
                    <item row="1" column="0">
                        <widget class="QDoubleSpinBox" name="dsb_x">
                            <property name="minimum">
                                <double>-10000000000000000000000.000000000000000</double>
                            </property>
                            <property name="maximum">
                                <double>10000000000000000000000.000000000000000</double>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="1">
                        <widget class="QDoubleSpinBox" name="dsb_y">
                            <property name="minimum">
                                <double>-10000000000000000000000.000000000000000</double>
                            </property>
                            <property name="maximum">
                                <double>10000000000000000000000.000000000000000</double>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="2">
                        <widget class="QDoubleSpinBox" name="dsb_z">
                            <property name="minimum">
                                <double>-10000000000000000000000.000000000000000</double>
                            </property>
                            <property name="maximum">
                                <double>10000000000000000000000.000000000000000</double>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="0">
                        <widget class="QLabel" name="label">
                            <property name="text">
                                <string>X-Verschiebung [m]</string>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignCenter</set>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="1">
                        <widget class="QLabel" name="label_2">
                            <property name="text">
                                <string>Y-Verschiebung [m]</string>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignCenter</set>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="2">
                        <widget class="QLabel" name="label_3">
                            <property name="text">
                                <string>Z-Verschieben [m]</string>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignCenter</set>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="FileSelector" name="widget_file_selector" native="true"/>
            </item>
            <item>
                <widget class="Progressbar" name="widget_progress_bar" native="true"/>
            </item>
            <item>
                <widget class="QDialogButtonBox" name="buttonBox">
                    <property name="standardButtons">
                        <set>QDialogButtonBox::Apply|QDialogButtonBox::Close</set>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <customwidgets>
        <customwidget>
            <class>FileSelector</class>
            <extends>QWidget</extends>
            <header>som_gui.module.util.ui</header>
            <container>1</container>
        </customwidget>
        <customwidget>
            <class>Progressbar</class>
            <extends>QWidget</extends>
            <header>som_gui.module.util.ui</header>
            <container>1</container>
        </customwidget>
    </customwidgets>
    <resources/>
    <connections/>
</ui>
